### PR TITLE
[Lint](clean): Remove some warnings (3)

### PR DESCRIPTION
**Pull request by alexandre-merle**
- Remove some warnings in lib/api files

### CODE REVIEW

File: lib/api/objectPutACL.js [hunk 1]
```diff
@@ -157,12 +154,13 @@ export default function objectPutACL(authInfo, request, log, cb) {
...
                             return next(errors.InvalidArgument);
                         }
-                        usersIdentifiedByGroup.push({
+                        return usersIdentifiedByGroup.push({
                             identifier: grantee.URI[0],
                             grantType: permission,
                             userIDType: 'uri',
                         });
                     }
+                    return undefined;
```
Comment (rahulreddy) [inline lib/api/objectPutACL.js]: Just `return;` would work, right.

**Comment by rahulreddy** [comment]
👍

**Comment by ThibaultRiviere** [comment]
:+1:

File: lib/api/objectPutACL.js (hunk 1 above)
Comment (alexandre-merle) [inline lib/api/objectPutACL.js]: yes :)

Comment (alexandre-merle) [inline lib/api/objectPutACL.js]: We need to return a value in fact, it's a linter warning too :)

Comment (rahulreddy) [inline lib/api/objectPutACL.js]: Ah, never mind then.

**Comment by LaurenSpiegel** [comment]
Thanks, @alexandre-merle!  👍

**Pull request merged**